playbooks/install-ambari-2_6.yml

```diff
@@ -15,12 +15,16 @@
   become: true
   tasks:
     - include_tasks: ../roles/ambari-server/tasks/install-{{ ansible_os_family }}.yml
-    - import_tasks: ../roles/ambari-server/tasks/setup.yml
+    - import_tasks: ../roles/ambari-server/tasks/sudo.yml
```

Comment:
although we setting the sudoers... should we use become_user for every other tasks or playbook with remote_ambari_user? (like for ambari server start etc.)

Reply:
Connecting to hosts as non-root user and becoming another non-root user [poses problems](http://docs.ansible.com/ansible/latest/become.html#becoming-an-unprivileged-user).  There are various solutions suggested in the docs, but the only one that worked for me on all hosts (CentOS, Debian, Ubuntu) is connecting as root.  If that's acceptable, I can make this change (set `ansible_ssh_user=root` and add `become_user`).